### PR TITLE
Updated README.md file

### PR DESCRIPTION
**Pull request by jferas**

### CODE REVIEW

**Comment by jferas** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/1?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #2** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/2?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #5** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/5?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #8** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/8?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #12** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/12?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/12?utm_source=stack-comment).

**Review by alexkeating — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,159 +1,93 @@
...
-slither-args: --filter-paths "./lib|./test" --exclude naming-convention,solc-version
+```bash
+scopelint check
 ```
 
-This means Slither is not run on the `lib` or `test` folders, and the [`naming-convention`](https://github.com/crytic/slither/wiki/Detector-Documentation#conformance-to-solidity-naming-conventions) and [solc-version](https://github.com/crytic/slither/wiki/Detector-Documentation#incorrect-versions-of-solidity) checks are disabled.
+#### Scopelint spec compatibility
 
-This `slither-args` field is where you can change the Slither configuration for your project, and the defaults above can of course be changed.
+Some tests will not show up when running `scopelint spec` because the methods they are testing are inherited in the `RadworksGovernor`. In order to get an accurate picture of the tests with `scopelint spec` add an explicit `propose` method to the `RadworksGovernor`. It should look like this:
```
Comment (wildmolasses) [inline README.md:49]: Currently this is the extent of what i see from `scopelint spec`. Do you see the same thing?
Screenshot 2023-11-23 at 10 52 33 PM

File: README.md [hunk 2]
```diff
@@ -1,159 +1,93 @@
...
-[^naming-convention]:
-    A rigorous test naming convention is important for ensuring that tests are easy to understand and maintain, while also making filtering much easier.
-    For example, one benefit is filtering out all reverting tests when generating gas reports.
+### Testing issues
 
-[^script-abi]: Limiting scripts to a single public method makes it easier to understand a script's purpose, and facilitates composability of simple, atomic scripts.
-[^sarif]:
-    [SARIF](https://sarifweb.azurewebsites.net/) (Static Analysis Results Interchange Format) is an industry standard for static analysis results.
-    You can read learn more about SARIF [here](https://github.com/microsoft/sarif-tutorials) and read about GitHub's SARIF support [here](https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning).
+This repo heavily leverages fuzz fork tests causing a significant number of RPC requests to be made. We leverage caching to minimize the number of RPC calls after the tests are run for the first time, but running these tests for the first may cause timeouts and consume a significant number of RPC calls.
```
Comment (wildmolasses) [inline README.md:93]:
```suggestion
This repo heavily leverages fuzz fork tests causing a significant number of RPC requests to be made. We leverage caching to minimize the number of RPC calls after the tests are run for the first time, but running these tests for the first time may cause timeouts and consume a significant number of RPC calls.
```

Comment (jferas) [reply README.md]: Done: 212a699ff89abe0e05628e549fe466b13d9dcb53

File: README.md (hunk 1 above)
Comment (wildmolasses) [reply README.md]: cool -- shouldn't it still show more tests though, the ones unrelated to `propose`?  Anyway, don't let this block merge, if it's an issue in the future then we can revise.

**Review by wildmolasses — APPROVED** [review]
lgtm, mergeable

File: README.md (hunk 1 above)
Comment (alexkeating) [reply README.md]: At this point it the codebase it doesn't look like it should. It finds the tests to show based on the name of the test contracts. Right now the only test contracts with tests are `Propose` and `Constructor`

**Pull request merged**